hedera-node/src/test/java/com/hedera/services/contracts/execution/HederaMessageCallProcessorTest.java

```diff
@@ -60,6 +60,10 @@ class HederaMessageCallProcessorTest {
 	private static final Address HTS_PRECOMPILE_ADDRESS = Address.fromHexString(HTS_PRECOMPILE_ADDRESS_STRING);
 	private static final Address RECIPIENT_ADDRESS = Address.fromHexString("0xcafecafe01");
 	private static final Address SENDER_ADDRESS = Address.fromHexString("0xcafecafe02");
+	private static final PrecompiledContract.PrecompileContractResult NO_RESULT = new PrecompiledContract.PrecompileContractResult(
+			Bytes.EMPTY, true, MessageFrame.State.COMPLETED_FAILED, Optional.empty());
+	private static final PrecompiledContract.PrecompileContractResult RESULT = new PrecompiledContract.PrecompileContractResult(
+			Bytes.of(1), true, MessageFrame.State.COMPLETED_FAILED, Optional.empty());
```

Comment:
Q: Is this supposed to be a successful completion or indeed a failed completion based on some non empty result?